README.adoc

```diff
@@ -229,12 +242,31 @@ message in the browser. Otherwise, this method returns the cached property value
 You successfully set up your microservice to have fault tolerance capability.
 
 
+== Enabling metrics for the fault tolerance methods
+
+server.xml
+[source, xml, linenums, role='code_column']
+----
+include::finish/src/main/liberty/config/server.xml[]
+----
+
+MicroProfile Fault Tolerance integrates with MicroProfile Metrics to provide metrics for the annotated fault tolerance methods.
+When both the [hotspot=mpFaultTolerance]`mpFaultTolerance` and the [hotspot=mpMetrics]`mpMetrics` features are included in the [hotspot]`server.xml` configuration file,
+the `@Fallback` fault tolerance annotation supplies metrics that count the total number of annotated method invocations and failed invocations,
+as well, the total number of the fallback method calls.
```

Comment:
Change to:
"When both the `mpFaultTolerance` and the `mpMetrics` features are included in the `server.xml` configuration file, the `@Fallback` fault tolerance annotation provides metrics that count the following things: the total number of annotated method invocations, the total number of failed annotated method invocations, and the total number of the fallback method calls."

Reply:
Updated